README.md

```diff
@@ -86,6 +86,13 @@ libraries require only one of these or are dependency-free.
 * [Consul](https://consul.io/)
 * [Runit](http://smarden.org/runit/), which includes chpst
 
+Many P2 binaries expect to be able to reference the `p2-exec` binary.
```

Comment:
going to be precise: s/reference/invoke/, and add "ideally by knowing its full path"